packages/esm-ward-app/src/ward-workspace/ward-patient-notes/notes.style.scss

```diff
@@ -0,0 +1,6 @@
+@import '~@openmrs/esm-styleguide/src/vars';
+
+.workspaceContainer {
+  width: calc(100% - 2rem);
+  min-height: var(--desktop-workspace-window-height);
+}
```

Comment:
The width issue needs to be fixed in the workspace container in the core. Please don't add these stylings here, else after the fix is made there, it will break here.

Reply:
Also the height styling isn't necessary. If anything is needed for height, it will be `height: 100%`.